java/graphscope-sdk/src/main/java/io/graphscope/utils/AppBaseParser.java

```diff
@@ -50,6 +50,8 @@ private static void loadClassAndParse(String className) {
                         return;
                     }
                     System.out.println("TypeParams: " + typeParams[0].getTypeName());
+                    Class<?> ctxType = (Class<?>) typeParams[1];
+                    System.out.println("ContextType:" + ctxType.getSuperclass().getName());
```

Comment:
I suggest to leverage some logging facilities, e.g., slf4j, for logging in java sdk, instead of `System.out.println`.

Reply:
This is used by https://github.com/zhanglei1949/GraphScope/blob/main/python/graphscope/analytical/app/java_app.py#L67 , and is not actually a log.
So maybe ```System.out.println``` is ok ? Cause it make no difference to python client when  parsing these messages.